src/Kovan-DssSpell.sol

```diff
@@ -31,6 +31,37 @@ interface LerpFabLike {
     function count() external returns (uint256);
 }
```

Comment:
I think the `LerpFabLike` interface can be removed.

Reply:
Fixed.